sparse/lib/systemd/system/adbd.service

```diff
@@ -1,5 +1,8 @@
 [Unit]
 Description=Android Debug Bridge Daemon
+
 [Service]
 Environment=PATH=/sbin:/vendor/bin:/system/sbin:/system/bin:/system/xbin
+ExecStartPre=/usr/sbin/adbd-functionfs.sh
 ExecStart=/sbin/adbd
+ExecStopPost=/bin/sleep 1 ; /bin/umount adb
```

Comment:
Yep the ; works as per documentation. However the ExecStopPost not really. Seems it still failed now and then. Maybe file handles not completely released yet or something. With a sleep it works always.( Or at least I did not see sporadic failures anymore)